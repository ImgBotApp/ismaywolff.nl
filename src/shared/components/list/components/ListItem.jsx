import styled from 'styled-components'

// prettier-ignore
const ListItem = styled.li`
  list-style-type: none;
`

export default ListItem
