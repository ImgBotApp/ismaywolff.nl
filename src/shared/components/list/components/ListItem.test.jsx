import 'jest-styled-components'
import React from 'react'
import { shallow } from 'enzyme'
import ListItem from './ListItem'

describe('<ListItem />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<ListItem />)
    expect(wrapper).toMatchSnapshot()
  })
})
