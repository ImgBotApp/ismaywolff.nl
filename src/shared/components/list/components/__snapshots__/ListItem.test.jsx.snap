// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListItem /> renders correctly 1`] = `
.c0 {
  list-style-type: none;
}

<li
  className="c0"
/>
`;
